src/SearchClient.php

```diff
@@ -11,12 +11,12 @@
 
 namespace Algolia\AlgoliaSearch\Helper;
 
-class SearchClient extends \Algolia\AlgoliaSearch\SearchClient
+final class SearchClient extends \Algolia\AlgoliaSearch\SearchClient
 {
     /**
      * Creates a new instance of the Search Client.
      *
-     * @param string $indexName
+     * @param mixed $indexName
```

Comment:
Why this should be mixed?

Reply:
Because in `Algolia\AlgoliaSearch\SearchClient::initIndex()` there is no documentation on what should be $indexName, PhpStan throw me an error because of inheritance.

Reply:
This is the error :
``` Parameter #1 $indexName (string) of method
Algolia\AlgoliaSearch\Helper\SearchClient::initIndex() should be
contravariant with parameter $indexName (mixed) of method
Algolia\AlgoliaSearch\SearchClient::initIndex() 
```

Reply:
Let's ignore this error for now. The problem is the missing documentation on the parent class.

Reply:
Yes.